todo.css

```diff
@@ -0,0 +1,36 @@
+body {
+  background-color: #FAECA9
+}
+
+.topBar {
+  font-family: sans-serif;
+  font-size: 1em;
+  text-align: center;
+  color: white;
+  position: fixed;
+  top:-3%;
```

Comment:
this shouldn't be needed. you should put `padding: 0; margin: 0` on `body` instead.